modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/SubsystemControlCell.scala

```diff
@@ -3,6 +3,8 @@
 
```

Comment:
Note all the changes in this file were just to help me track / understand, so you can ignore as they'll all be removed.